examples/tts/conf/fastpitch_align_ipa.yaml

```diff
@@ -0,0 +1,246 @@
+# This config contains the default values for training a FastPitch model with aligner.
+# If you want to train a model on other dataset, you can change config values according to your dataset.
+# Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+train_dataset: ???
+validation_datasets: ???
+sup_data_path: ???
+sup_data_types: [ "align_prior_matrix", "pitch" ]
+
+# Default values from librosa.pyin
+pitch_fmin: 65.40639132514966
+pitch_fmax: 2093.004522404789
+
+# LJSpeech stats (per frame), train (these values depend on pitch_fmin and pitch_fmax)
+pitch_mean: 212.35873413085938
+pitch_std: 68.52806091308594
+
+# Default values for dataset with sample_rate=22050
+sample_rate: 22050
+n_mel_channels: 80
+n_window_size: 1024
+n_window_stride: 256
+n_fft: 1024
+lowfreq: 0
+highfreq: 8000
+window: hann
+
+phoneme_dict_path: "scripts/tts_dataset_files/ipa_cmudict-0.7b_nv22.06.txt"
+heteronyms_path: "scripts/tts_dataset_files/heteronyms-052722"
+whitelist_path: "nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv"
```

Comment:
IPA and ARP symbols weren't normalized because LJSpeech whitelist is specified. For a different dataset, the default normalization could alter phoneme symbols. It's safer to pass phonemes in square brackets.

Reply:
IPA is performed after normalization, so this should be fine as is, I believe?